packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts

```diff
@@ -28,6 +28,7 @@ export const BreadcrumbItemStyles = css`
     .listitem {
         display: flex;
         align-items: center;
+        width: max-content;
```

Comment:
Is this a new addition?

Reply:
yeah, it's an addition to fixing the non-href item. I noticed from running the stackblitz example that if the window was small enough, it would make the item, which is not an href, wrap itself.
![image](https://user-images.githubusercontent.com/37851220/115315603-86f88900-a12c-11eb-830c-21fd87436486.png)